src/main/java/frc/robot/RobotContainer.java

```diff
@@ -351,31 +353,9 @@ private void configureButtonBindings() {
   }
 
   private void configureUniversalControls(CommandXboxController controller) {
-    new Trigger(() -> DriverStation.getMatchTime() < 30)
-        .onTrue(
-            Commands.runOnce(
-                    () -> {
-                      controller.getHID().setRumble(GenericHID.RumbleType.kBothRumble, 0.5);
-                    })
-                .andThen(Commands.waitSeconds(1))
-                .andThen(
-                    Commands.runOnce(
-                        () -> {
-                          controller.getHID().setRumble(GenericHID.RumbleType.kBothRumble, 0);
-                        })));
-
-    new Trigger(() -> DriverStation.getMatchTime() < 15)
-        .onTrue(
-            Commands.runOnce(
-                    () -> {
-                      controller.getHID().setRumble(GenericHID.RumbleType.kBothRumble, 0.5);
-                    })
-                .andThen(Commands.waitSeconds(1))
-                .andThen(
-                    Commands.runOnce(
-                        () -> {
-                          controller.getHID().setRumble(GenericHID.RumbleType.kBothRumble, 0);
-                        })));
+    new TimeTrigger(40).or(new TimeTrigger(20))
```

Comment:
this is wrong, it will be true as long as the time is < 40, which means the 20s trigger does nother